client/src/components/header.tsx

```diff
@@ -75,6 +76,10 @@ function Header(): JSX.Element {
     setOpen(true);
   };
 
+  const handleClickHome = () => {
+    window.location.href="/admin";
```

Comment:
will this work with the actual app? wouldn't href be something like https://v2.mentorpal.org/admin/ instead?